apps/mobile-wallet/ios/Podfile.lock

```diff
@@ -1618,7 +1618,7 @@ PODS:
     - React-Core
   - RNFlashList (1.6.4):
```

Comment:
Some screens are missing a way to go back. The button is there but hidden. I can click top left and go back:
1. Receive screen, step 1
2. Send screen, step 1

<img width="350" alt="image" src="https://github.com/user-attachments/assets/d4186db5-d143-486a-a403-8e12df0718cf">
<img width="350" alt="image" src="https://github.com/user-attachments/assets/9fb19aa1-7de6-4664-aece-d0f91c1b343f">

Reply:
mhhh used to work too, strange